tests/integration/api/test_online_enable.py

```diff
@@ -54,10 +50,11 @@ async def app_service_fixture(persistent):
                         return
                 except requests.exceptions.ConnectionError:
                     pass
+                logger.info("waiting for service to be started...")
                 await asyncio.sleep(1)
             raise TimeoutError("service did not start")
         finally:
-            proc.terminate()
+            proc.kill()
```

Comment:
What's the difference between terminate and kill?

Reply:
Terminate sends SIGTERM and kill sends SIGKILL: https://docs.python.org/3/library/subprocess.html#subprocess.Popen.terminate

When testing I found that `terminate()` doesn't end the process reliably, probably because uvicorn has some special handling for that signal. I don't think we need graceful exit for this process so `kill()` should be fine.